packages/app/provider/font/index.tsx

```diff
@@ -13,7 +20,15 @@ export const FontProvider = ({ children }: { children: React.ReactNode }) => {
     'Inter-Bold': Inter_700Bold,
   })
 
-  if (!fontsLoaded) {
+  const [ralewayLoaded] = useFonts({
```

Comment:
what about using the `.ttf` file instead of the library? This way it should work for the web as well.